src/Service/Lambda/README.md

```diff
@@ -10,3 +10,29 @@ An API client for Lambda.
 ```cli
 composer require async-aws/lambda
 ```
+
+## Usage
+
+````php
+$client = new LambdaClient([
+    Configuration::OPTION_REGION => 'us-east-1',
+    Configuration::OPTION_ACCESS_KEY_ID => 'Access key ID',
+    Configuration::OPTION_SECRET_ACCESS_KEY => 'Access key Secret',
```

Comment:
Removed those 2 lines: writing secret is a bad practice, (it can be commited and exposed). 
and authentication process is documented in https://github.com/async-aws/aws/blob/master/docs/authentication.md

Reply:
I agree, it was for the example. In my implementation it was loading ENV VARS. 
However if we omit the AWS authentication part in the example I fear new comers would lose time implementing `async-aws/lambda` trying to deduce how to connect to AWS.

What do you think ?

Reply:
maybe we should promote use of env variables in that case.
```cli
export AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
export AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
```
```php
...
```